test/dpul_collections_web/features/search_bar_test.exs

```diff
@@ -28,4 +28,17 @@ defmodule DpulCollectionsWeb.Features.SearchBarTest do
     |> click_button("Search")
     |> assert_has("#item-counter", text: "1 - 1 of 1")
   end
+
+  test "renders facet with states", %{conn: conn} do
```

Comment:
Could this test go in `search_live_test`? It's really testing the search results page, as opposed to the search bar component.